v1/payroll/employee_time_worked/post.md

```diff
@@ -15,29 +15,82 @@ may be included for the employee.
 
 ### Request body
 
-| Name        | Type     | Data Type                      | Description                                             |
-|-------------|----------|--------------------------------|---------------------------------------------------------|
-| employee_id | required | uuid (v4)                      | The unique identifier for the employee                  |
-| time_worked | required | array of `Time Worked` entries | An array that contains at least one `Time Worked` entry |
+| Name                        | Type     | Data Type                                     | Description                                                                     |
+|-----------------------------|----------|-----------------------------------------------|---------------------------------------------------------------------------------|
+| employee_id                 | required | uuid (v4)                                     | The unique identifier for the employee                                          |
+| time_worked                 | required | array of `Time Worked` entries                | An array that contains at least one `Time Worked` entry                         |
+| reset                       | optional | boolean                                       | When set to true will destroy all previous time worked entries on open payrolls |
```

Comment:
we can mention that  currently we have set it to be true.
we did this to remove work load from Ameego

Reply:
@atuljha19 when you say "it is currently set to true" do you mean that if the value is absent we default to true?

Reply:
yes!